packages/components/src/components/install-guide/record-instructions/Ruby.vue

```diff
@@ -1,69 +1,91 @@
 <template>
   <section>
-    <p class="mb20 bold">
-      When you run your Ruby code with the <tt>appmap</tt> gem in your bundle, AppMap will be
-      enabled for recording. By default, the <tt>appmap</tt> gem is only enabled in the "test" and
-      "development" environments.
+    <p>
+      When you run your Ruby code with the <code class="inline">appmap</code> gem in your bundle,
+      AppMap will be enabled for recording. By default, the <code class="inline">appmap</code> gem
+      is only enabled in the "test" and "development" environments.
     </p>
+    <br />
     <p>
       Before you run your app with AppMap, be sure and stop or disable Spring. You can stop Spring
-      with the command <tt>spring stop</tt>, or you can run your Ruby program with the environment
-      variable <tt>DISABLE_SPRING=true</tt>.
+      with the command <code class="inline">spring stop</code>, or you can run your Ruby program
+      with the environment variable <code class="inline">DISABLE_SPRING=true</code>.
     </p>
+    <br />
+    <h2>Choose the best recording method for this project</h2>
+    <br />
     <template v-if="testFramework">
-      <h3>Tests recording</h3>
-      <p>
-        When you run your {{ testFramework.name }} tests with AppMap enabled, an AppMap will be
-        created for each test.
-      </p>
-      <template v-if="testFramework.name.toLowerCase() == 'minitest'">
-        Run Minitest tests:
-        <v-code-snippet clipboard-text="DISABLE_SPRING=true rails test" />
-      </template>
-      <template v-if="testFramework.name.toLowerCase() == 'rspec'">
-        Run Rspec tests:
-        <v-code-snippet clipboard-text="DISABLE_SPRING=true rspec" />
-      </template>
-      <p>
-        For more information, visit
-        <a href="https://appmap.io/docs/reference/appmap-ruby.html#tests-recording" target="_blank"
-          >AppMap docs - Ruby Tests recording</a
-        >.
-      </p>
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><TestsIcon /></i>Tests recording<span class="recommended-badge"
+            >recommended</span
+          >
+        </h3>
+        <p>
+          When you run your {{ testFramework.name }} tests with AppMap enabled, an AppMap will be
+          created for each test.
+        </p>
+        <p>
+          <template v-if="testFramework.name.toLowerCase() == 'minitest'">
+            Run Minitest tests:
```

Comment:
Would be nice to also only show the instructions for the test framework they're using (if they're only using one of them).

Reply:
That’s what this section does.

Reply:
Should be triple equals though - I think.

Reply:
Oh I missed the `v-if`, thanks.